clouddq/integration/dataplex/dataplex_client.py

```diff
@@ -234,31 +249,7 @@ def delete_dataplex_task(
         )
         return response
 
-    def get_dataplex_iam_permissions(
```

Comment:
Is that clean-up intended?

I can still see some references to that function in the clouddq_dataplex.py

def get_iam_permissions(self) -> list:
        body = {"resource": "dataplex", "permissions": ["roles/dataproc.worker"]}
        return self._client.**get_dataplex_iam_permissions**(
            body=body,
        )

Please doublecheck.

Reply:
oh right thanks! That's not actually used anywhere so I will delete that as well.

Reply:
perfect, thanks!